pkg/gcpclient/client.go

```diff
@@ -97,17 +101,44 @@ func NewClient(projectName string, authJSON []byte) (Client, error) {
 		return nil, fmt.Errorf("gcpclient.cloudBillingClient.NewService %v", err)
 	}
 
+	computeService, err := compute.NewService(ctx, option.WithCredentials(creds))
+	if err != nil {
+		return nil, fmt.Errorf("gcpclient.compute.NewService %v", err)
+	}
+
 	return &gcpClient{
 		projectName:                projectName,
 		creds:                      creds,
 		cloudResourceManagerClient: cloudResourceManagerClient,
 		iamClient:                  iamClient,
 		serviceManagmentClient:     serviceManagmentClient,
 		cloudBillingClient:         cloudBillingClient,
+		computeClient:              computeService,
 		credentials:                creds,
 	}, nil
 }
 
+// ListAvilibilityZones returns a map of all avilibility zones a project has access to
+// where the key is the region and the values is a list of zones
+func (c *gcpClient) ListAvilibilityZones(projectID, region string) (map[string][]string, error) {
```

Comment:
If I understand this function correctly it only lists the zones of the given regions. So, given input region "x", with matching zones "z1" and "z2" it will return
```
{
"x": ["z1", "z2"]
}
```
why not just return an array `["z1", "z2"]`?

Reply:
The reason I put this into a map was to support additional regions/zones in the future , but if its not necessary now I can remove it.